app/script_variant.py

```diff
@@ -29,6 +29,15 @@ def read_dictionary(file_name):
 JAPANESE_LOCATION_DICTIONARY = read_dictionary('jp_location_dict.txt')
 
 
+def is_kanji(word):
+    """
+    Returns whether word is kanji or not.
+    [\u3400-\u9fff]: kanji
+    TODO: update this range (some of kanji characters out of this range.)
+    """
+    return re.match(ur'([\u3400-\u9fff])', word)
+
+
 def romanize_japanese_name_by_name_dict(word):
     """
     This method romanizes japanese name by using name dictionary.
```

Comment:
> If word isn't found in dictionary, this method doesn't apply romanize.

This is not exactly true. You romanize hiragana words not in dictionary, right? Please fix.